src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -32,6 +32,7 @@
 
 namespace Voron.Impl.Journal
 {
+    [SuppressMessage("CancellationToken", "RDB0007:CancellationToken must be a last argument")]
```

Comment:
weird?

Reply:
`Voron` project previously didn't have the analysis enabled. After enabling it, all the rules were enforced. As this is infrastructural code I left as it was. I can refactor to satisfy the rule.

Reply:
leave it as is